src/controllers/restful/PaymentController.ts

```diff
@@ -385,6 +386,9 @@ export function getErrorMessageCtrlFromPagoPaError(
           );
         }
       }
+      logger.debug(
```

Comment:
can you make this a warning? I'd like to see these messages also when we run in production, thanks

Reply:
Really it's not a warning.
The first 4 cases are just specified to App because it handle them in specific ways.
The "else" case is "any other errors". But most common and normal errors will be catched by "else".

Reply:
I don't understand what you suggest to do - what I'd like to do is to log `faultCode`s and `faultDescription`s that we don't handle and later see whether there's something interesting we should handle

Reply:
ok